Formula/mos.rb

```diff
@@ -44,6 +46,11 @@ def install
   end
 
   test do
-    system bin/"mos", "version"
+    # Save license server key
```

Comment:
an odd choice of test command. not everyone has credentials set up for this to work.

Reply:
Thanks for the review @rojer.

The test I wrote works by saving a random license server key, and then checking whether it has been stored in the auth.json file. Since these tests are isolated from the user, it shouldn't depend on whether they've got the credentials set up (I don't for instance 😉) , and it shouldn't affect their files either.

auth.json contents:
```
{
  "license_server_access": [
    {
      "server": "https://license.mongoose-os.com",
      "key": "example"
    }
  ]
}
```

However, you know how this command works better than I do, so let me know if I've missed something and I'd be more than happy to change the test. I will however certainly update the comments to make it easier to understand what's happening.

Reply:
i see. still, i think this is not quite the right approach here.
"mos version" is ok as a smoke test, i.e. that a binary runs at all, i think that is sufficient. we may also want to test mac-specific functionality but rest of the testing belongs elsewhere - expressed as ago unit test or integration test (we don't have those yet).

Reply:
Thanks @rojer for your response.

I put the test back to `mos version`, but as an extra, I made it verify that the output is what it should be.